pom.xml

```diff
@@ -138,7 +138,7 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-surefire-plugin</artifactId>
-                    <version>3.0.0-M5</version>
+                    <version>3.5.0</version>
```

Comment:
The newest version right now is 3.5.4. Might be worth to update

Reply:
I've updated the plugin version accordingly.